src/python/pants/option/options_bootstrapper_test.py

```diff
@@ -327,72 +327,16 @@ def parse_options(*args: str) -> OptionValueContainer:
     def test_bootstrap_options_passthrough_dup_ignored(self) -> None:
         def parse_options(*args: str) -> OptionValueContainer:
             full_args = [*args, *self._config_path(None)]
-            return (
-                OptionsBootstrapper.create(env={}, args=full_args, allow_pantsrc=False)
-                .get_bootstrap_options()
-                .for_global_scope()
-            )
+            return OptionsBootstrapper.create(
+                env={}, args=full_args, allow_pantsrc=False
+            ).bootstrap_options.for_global_scope()
 
         vals = parse_options("main", "args", "-lwarn", "--", "-lerror")
         assert LogLevel.WARN == vals.level
 
         vals = parse_options("main", "args", "--", "-lerror")
         assert LogLevel.INFO == vals.level
 
-    def test_bootstrap_options_explicit_config_path(self) -> None:
```

Comment:
This was testing the bespoke `--pants-config-files` parsing logic, which was deleted. However since this option is consulted in the rust bootstrap sequence, I added extra testing for that there (see below). That new test is less meticulous than this one, however, since it's read via regular option parsing code that is heavily tested already.